src/main/java/hibernate/tournament/model/MatchDto.java

```diff
@@ -0,0 +1,39 @@
+package hibernate.tournament.model;
+
+import lombok.*;
```

Comment:
Avoid importing all from the package.
You force developers to use API that may be not needed.